clang/lib/DPCT/SaveNewFiles.cpp

```diff
@@ -199,6 +196,26 @@ void rewriteFileName(std::string &FileName, const std::string &FullPathName) {
 
 static std::vector<std::string> FilesNotInCompilationDB;
 
+std::map<std::string, std::string> OutFilePath2InFilePath;
+
+static bool checkOverwriteAndWarn(StringRef OutFilePath, StringRef InFilePath) {
+  auto SrcFilePath = OutFilePath2InFilePath.find(OutFilePath.str());
+
+  bool Overwrites = false;
+  // Make sure that the output file corresponds to a single and unique input
+  // file.
+  if (SrcFilePath != OutFilePath2InFilePath.end() &&
+      SrcFilePath->second != InFilePath) {
+    llvm::errs() << "[WARNING]: The output file of '" << InFilePath << "' and '"
+                 << SrcFilePath->second << "' have same name '" << OutFilePath
+                 << "'. To avoid overwrite, the migration of '" << InFilePath
+                 << "' is skipped. Please change the output file extension "
+                    "with option '--sycl-file-extension'.\n";
```

Comment:
Do we need use `getNL()` instead of '\n'?

Reply:
Is it better then `\n`?

I tried searching for documentation of `getNL()`, but I couldn't find any!

Reply:
Done!